<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiverGUI</class>
 <widget class="QMainWindow" name="ReceiverGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReceiverGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>1</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border: 2px solid gray;
}
QGroupBox::title{
	background-color: transparent;
    subcontrol-position: top left;
    padding: 2 5px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSlider" name="sl_qf">
     <property name="geometry">
      <rect>
       <x>113</x>
       <y>34</y>
       <width>160</width>
       <height>29</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
     <property name="sliderPosition">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>JPEG q.f.</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_CTA">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Enable CTA</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>107</x>
       <y>55</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>271</x>
       <y>55</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>86</y>
      <width>311</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border: 2px solid gray;
}
QGroupBox::title{
	background-color: transparent;
    subcontrol-position: top left;
    padding: 2 5px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radio_ATC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Enable ATC</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ck_coding">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>122</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Entropy coding</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ck_showRec">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>152</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Show reconstruction</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>46</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>det. thr.</string>
     </property>
    </widget>
    <widget class="QSlider" name="sl_thr">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>39</y>
       <width>160</width>
       <height>29</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>30</number>
     </property>
     <property name="maximum">
      <number>120</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
     <property name="sliderPosition">
      <number>60</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>60</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>30</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>259</x>
       <y>60</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>120</string>
     </property>
    </widget>
    <widget class="QSlider" name="sl_maxFeats">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>78</y>
       <width>160</width>
       <height>29</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>25</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="singleStep">
      <number>50</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="sliderPosition">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>99</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>25</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>258</x>
       <y>99</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>500</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>85</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>max feats.</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_matchThr">
     <property name="geometry">
      <rect>
       <x>51</x>
       <y>186</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>match thr.</string>
     </property>
    </widget>
    <widget class="QSlider" name="sl_matchThr">
     <property name="geometry">
      <rect>
       <x>129</x>
       <y>179</y>
       <width>160</width>
       <height>29</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>200</number>
     </property>
     <property name="sliderPosition">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="lab_matchThr0">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>200</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_matchThr255">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>200</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>255</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bt_start">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>196</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border: 2px solid gray;
}
QGroupBox::title{
	background-color: transparent;
    subcontrol-position: top left;
    padding: 2 5px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Transmission parameters</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Device</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>83</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Delay</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="tx_radioDev">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>/dev/ttyUSB0
</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="tx_delay">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>75</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="plainText">
      <string>10000</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>322</y>
      <width>521</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
	border: 2px solid gray;
}
QGroupBox::title{
	background-color: transparent;
    subcontrol-position: top left;
    padding: 2 5px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>3</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Received data</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OBJECT:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>FRAME RATE:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BANDWIDTH:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lab_bw">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>bw</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_recObj">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>obj</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_fps">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>fps</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
